src/main/java/com/vladyslavpalamarchuk/trackmycoin/service/MessageHandlerService.java

```diff
@@ -0,0 +1,44 @@
+package com.vladyslavpalamarchuk.trackmycoin.service;
+
+import com.vladyslavpalamarchuk.trackmycoin.adaptors.telegram.TelegramBotClient;
+import com.vladyslavpalamarchuk.trackmycoin.domain.Command;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.telegram.telegrambots.meta.api.methods.send.SendMessage;
+import org.telegram.telegrambots.meta.exceptions.TelegramApiException;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class MessageHandlerService {
+
+    @Autowired
+    private TelegramBotClient telegramBotClient;
+
+    private final List<Command> commands = List.of(Command.START , Command.INFO , Command.HELP);
```

Comment:
Please use strategy pattern

Reply:
should be like this
`commandProcessorsRegistry.getProcessor(Command).process(chatId)`